release/nightly_tests/many_nodes_tests/app_config.yaml

```diff
@@ -1,5 +1,5 @@
 base_image: "anyscale/ray-ml:pinned-nightly-py37"
-env_vars: {}
+env_vars: {"RAY_gcs_server_rpc_server_thread_num": "8"}
```

Comment:
Actually can we set this by default instead of overriding in a test? It's not good for test configs to diverge from prod.

Reply:
How should we select the thread num? (# of cpus) / 8? or just maybe 8 by default?

Reply:
8 seems fine as a sensible default